it/AssetsControllerISpec.scala

```diff
@@ -97,19 +139,50 @@ class AssetsControllerISpec extends IntegrationSpecBase {
         }
       }
     }
+
+    s"return ${Status.OK} and parse to xml when using V2" when {
+      allFilesFromEuV2.foreach { eachFile =>
+        s"file is ${eachFile.getName}" in {
+          val pathToFile           = eachFile.getCanonicalPath.split("/CR-for-NES-Services-V2/")(1).trim
+          val resultOfGettingAsset = await(buildClient(s"/assets/eu/outbound/CR-for-NES-Services-V2/$pathToFile").get())
+          resultOfGettingAsset.status shouldBe Status.OK
+
+          val sourceOfFile            = Source.fromFile(eachFile, "UTF-8")
+          val byteArrayStreamOfFile   = new ByteArrayInputStream(sourceOfFile.mkString.getBytes("UTF-8"))
+          val byeArrayStreamOfBody    = new ByteArrayInputStream(resultOfGettingAsset.body.getBytes("UTF-8"))
+          val fileFromDirectoryParsed = Try(XML.load(byteArrayStreamOfFile))
+
+          fileFromDirectoryParsed.get shouldBe XML.load(byeArrayStreamOfBody)
+
+          sourceOfFile.close()
+          byteArrayStreamOfFile.close()
+          byeArrayStreamOfBody.close()
+        }
+      }
+    }
   }
 
-  wsdlOperationsForFileNames.foreach {
-    case (fileName, wsdlOperationList) =>
-      s"$wsdlBaseUrl" when {
-        s"a request is made for $fileName" should {
-          wsdlOperationList.foreach { wsdlOperation =>
-            s"include the operation $wsdlOperation" in {
-              parseWsdlAndGetOperationsNames(wsdlBaseUrl + fileName) should contain(wsdlOperation)
+  def checkWsdlOperations(wsdls: Map[String, List[String]], wsdlBaseUrl: String): Unit = {
+    wsdls.foreach {
+      case (fileName, wsdlOperationList) =>
+        s"$wsdlBaseUrl" when {
+          s"a request is made for $fileName" should {
+            wsdlOperationList.foreach { wsdlOperation =>
+              s"include the operation $wsdlOperation" in {
+                parseWsdlAndGetOperationsNames(wsdlBaseUrl + fileName) should contain(wsdlOperation)
+              }
             }
           }
         }
-      }
+    }
+  }
+
+  def countAllWSDLandXSDFiles(files: Array[File]): Int = {
+    files.count(file => file.getName.contains(".xsd") || file.getName.contains(".wsdl"))
```

Comment:
We could use `.endsWith` instead of `.contains`

Reply:
Hi Vincent, yep that would be better - thanks